upup/pkg/fi/cloudup/awsup/aws_cloud.go

```diff
@@ -161,17 +171,25 @@ var awsCloudInstances map[string]AWSCloud = make(map[string]AWSCloud)
 func NewAWSCloud(region string, tags map[string]string) (AWSCloud, error) {
 	raw := awsCloudInstances[region]
 	if raw == nil {
-		c := &awsCloudImplementation{region: region}
+		c := &awsCloudImplementation{
+			region: region,
+			regionDelayers: &RegionDelayers{
+				delayerMap: make(map[string]*k8s_aws.CrossRequestRetryDelay),
+			},
+		}
 
 		config := aws.NewConfig().WithRegion(region)
 
-		// Add some logging of retries
-		config.Retryer = newLoggingRetryer(ClientMaxRetries)
-
 		// This avoids a confusing error message when we fail to get credentials
 		// e.g. https://github.com/kubernetes/kops/issues/605
 		config = config.WithCredentialsChainVerboseErrors(true)
 
+		// TODO Should we improve LoggingRetryer instead of what k8s core is using?
+		// TODO According to the aws api docs we will still be using the default retryer
+		// TODO and then we will have the k8s handlers that are acting as global retriers.
+		// TODO My thought is keep what we have and add the global handlers.
```

Comment:
Yes.  Can we demote this from a TODO to an explanation of what we're doing

Reply:
removing, as it does not really make sense.  I need to limit the paging, not worry about that